README.adoc

```diff
@@ -17,6 +17,55 @@ An end-to-end https://www.optaplanner.org/[OptaPlanner] web application ready to
 
 See https://www.youtube.com/watch?v=sOWC4qrXxFk[the video demo].
 
+== Quick Start
+
+Install Docker and run the following on the command line:
+
+[source,shell]
+----
+docker build -t rostering .
+docker run -p 8080:8080 --rm -it -e SPRING_PROFILES_ACTIVE=dev rostering
+----
+This takes a few minutes on the first time. The application will be automatically populated with test data.
+
```

Comment:
Probably want to say something like: "You can access the application on localhost:8080."
```suggestion
You can access the application on localhost:8080 once the application finish starting.
```